### PR TITLE
Bug 1826015: Align action button colors in multi/single column field

### PR DESCRIPTION
**Pull request by karthikjeeyar**
**Fixes**: https://issues.redhat.com/browse/ODC-3592

**Problem:** 
Align the action button with the colors provided by UX.

**Solution:**
Added colors for active/hover/disabled state of an action button in single/multi column fields.

**Screenshot:**

multicolumfield

singleColumnfield

Browser conformance:

- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

cc: @serenamarie125

### CODE REVIEW

**Comment by christianvogt** [comment]
What's the reason this isn't a pf icon button for which we would simply inherit their colors?

**Comment by karthikjeeyar** [comment]
> What's the reason this isn't a pf icon button for which we would simply inherit their colors?

It is a pf icon and just aligning with the hover/disable state colors.

**Comment by karthikjeeyar** [comment]
@christianvogt  I have added PF button(Plain) and now multicolumn field has keyboard accessibility and desired colors.

File: frontend/packages/console-shared/src/components/formik-fields/TextColumnField.tsx [hunk 1]
```diff
@@ -60,16 +69,17 @@ const TextColumnField: React.FC<TextColumnFieldProps> = ({
...
+                          isDisabled={disableDeleteRow}
                           onClick={() => {
-                            if (rowValues.length === 1) {
-                              return;
-                            }
                             arrayHelpers.remove(idx);
                           }}
-                        />
+                        >
+                          <MinusCircleIcon aria-hidden="true" />
```
Comment (christianvogt) [inline frontend/packages/console-shared/src/components/formik-fields/TextColumnField.tsx:81]: Not sure why you need aria-hidden.
```suggestion
                          <MinusCircleIcon />
```

File: frontend/packages/console-shared/src/components/formik-fields/TextColumnField.tsx [hunk 2]
```diff
@@ -60,16 +69,17 @@ const TextColumnField: React.FC<TextColumnFieldProps> = ({
                     </FlexItem>
                     {!isReadOnly && (
                       <FlexItem>
-                        <MinusCircleIcon
-                          aria-hidden="true"
-                          style={{ color: rowValues.length === 1 ? disabledColor.value : null }}
+                        <Button
```
Comment (christianvogt) [inline frontend/packages/console-shared/src/components/formik-fields/TextColumnField.tsx:72]: Delete or remove?
```suggestion
                        <Button aria-label="Delete"
```

File: frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx [hunk 3]
```diff
@@ -16,8 +15,16 @@ export interface MultiColumnFieldRowProps {
...
-      <MinusCircleIcon aria-hidden="true" onClick={onDelete} />
+    <div className={'odc-multi-column-field__col--button'}>
+      <Button
+        variant={ButtonVariant.plain}
+        type={ButtonType.button}
+        isInline
+        onClick={onDelete}
+        isDisabled={disableDeleteRow}
+      >
+        <MinusCircleIcon aria-hidden="true" onClick={onDelete} />
```
Comment (christianvogt) [inline frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx:26]:
```suggestion
        <MinusCircleIcon />
```

File: frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx [hunk 4]
```diff
@@ -16,8 +15,16 @@ export interface MultiColumnFieldRowProps {
 
 const minusCircleIcon = (onDelete: () => void, disableDeleteRow?: boolean, toolTip?: string) => {
   return (
-    <div className={cx('odc-multi-column-field__col--button', { 'is-disabled': disableDeleteRow })}>
-      <MinusCircleIcon aria-hidden="true" onClick={onDelete} />
+    <div className={'odc-multi-column-field__col--button'}>
+      <Button
```
Comment (christianvogt) [inline frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx:19]: Delete or remove?
```suggestion
      <Button aria-label="Delete"
```

File: frontend/packages/console-shared/src/components/formik-fields/TextColumnField.tsx (hunk 1 above)
Comment (karthikjeeyar) [reply frontend/packages/console-shared/src/components/formik-fields/TextColumnField.tsx]: removed it.

File: frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx (hunk 3 above)
Comment (karthikjeeyar) [reply frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx]: updated

**Pull request merged**